server/src/test/java/com/broadcom/lsp/cobol/TestModule.java

```diff
@@ -61,7 +62,7 @@ protected void configure() {
     bind(CopybookService.class).to(CopybookServiceImpl.class);
     bind(Communications.class).to(ServerCommunications.class);
     bind(TextDocumentService.class).to(CobolTextDocumentService.class);
-    bind(CobolTextRegistry.class).to(ZipTextRegistry.class);
+    bind(CobolTextRegistry.class).to(FolderTextRegistry.class);
```

Comment:
If we don't plan to use a zip file as a registry, we can delete the entire `ZipTextRegistry` class and `CobolTextHelper`.
In case we work only with a folder registry, we can read a file via `Files.readString(path)` in `FolderTextRegistry::processFile`, no needs to touch `InputStream` at all.

Reply:
Actually, I don't know do we need to have the ability to use `ZipTextRegistry`. In that case, maybe it is better to be able to detect by path what registry needs to be used here.

Reply:
We don't need the ZipTextRegistry anymore, so you may remove this class. Also, seems like we don't need this binding - we do not inject this object anywhere

Reply:
Got it